src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -241,7 +241,7 @@ protected void ValidateArgumentIsMutable(TexlBinding binding, TexlNode arg, IErr
 
         // This method is used for managing "x-ms-dynamic-values" and "x-ms-dynamic-schema" OpenApi extensions in connectors
         // https://learn.microsoft.com/en-us/connectors/custom-connectors/openapi-extensions
-        public virtual async Task<ConnectorSuggestions> GetConnectorSuggestionsAsync(FormulaValue[] knownParameters, int argPosition, CancellationToken cts)
+        public virtual async Task<ConnectorSuggestions> GetConnectorSuggestionsAsync(FormulaValue[] knownParameters, int argPosition, IServiceProvider services, CancellationToken cts)
```

Comment:
> IServiceProvider services, [](http://example.com/codeflow?start=133&length=27)

Can this break PA?

Reply:
we should be ok - PA doesn't implement this virtual.